idprovider/local/idprovider_test.go

```diff
@@ -239,9 +239,9 @@ func Test_YAML_Delete_Read(t *testing.T) {
 func Test_YAML_UpdateStorage(t *testing.T) {
 	t.Run("happy_empty_file", func(t *testing.T) {
 		// Setup: NewYAML with zero entries
-		emptyFile := contactstest.NewYAMLFileT(t)
-		fileWithTwoPeers := contactstest.NewYAMLFileT(t, peer1, peer2)
-		c, err := contactsyaml.New(emptyFile, walletBackend)
+		emptyFile := idprovidertest.NewIDProviderT(t)
+		fileWithTwoPeers := idprovidertest.NewIDProviderT(t, peer1, peer2)
```

Comment:
```suggestion
		fileWithTwoPeerIDs := idprovidertest.NewIDProviderT(t, peer1, peer2)
```

Using peer ID instead of peer. Other usage of the variable also needs to be updated.

Reply:
Done